ui/app/components/app/modals/customize-nonce/index.scss

```diff
@@ -0,0 +1,61 @@
+.customize-nonce-modal {
+  padding-left: 24px;
+  padding-right: 18px;
+  display: flex;
+  flex-flow: column nowrap;
+
+  &__main-header {
+    display: flex;
+    flex-flow: row;
+    justify-content: left;
+    align-items: left;
+    position: relative;
+    padding-top: 24px;
+  }
+
+  &__close {
+    @include H4;
+
+    color: #24292e;
+    background: none;
+    position: absolute;
```

Comment:
Using absolute positioning should be a last resort because it becomes notoriously hard to maintain upon iteration. 

Instead, you have the basis of what you need already. Below is a description of changes and some screenshots that should help get you where you want to be.

#### &__close (close button)
1. remove width and height properties. These properties control the size of the containing box, but not the size of the font which is being rendered (the @include H4 does that). The result is that your element tries to take up less space than what it contains.
2. Remove cursor: pointer, buttons already have this property
3. remove position: absolute, and right: 10px; -- when you do this it'll be borked! We'll fix it in the next step.
4. add `flex: 0` (this tells flex box that this element should take up as little space as possible :) )
5. Add `align-self: flex-start` this tells this element to ignore the `align-items` of the parent and align this element to the top of the row.

<details>
<summary>Screenshot of issue in first bullet </summary>

![image](https://user-images.githubusercontent.com/4448075/114750525-b0bf4380-9d19-11eb-9f9c-5b890ce6203a.png)


</details>

#### &__main-header
1. remove position: relative; 
2. Add align-items: center; (this centers the items in the flex row vertically)

#### title Typography component
1. Add a class name `&__main-title` or something similar
2. add `flex: 1` to this class, which tells flex box that this element should take up as much space as possible. 

Once these steps are complete you should have the desired result :)